### DISCUSSION TITLE
Permission scope for to retrieve release notes

### DISCUSSION THREAD

**Discussion by andormarkus** [renovatebot/renovate#10920] [Request Help] [answered]
Hi Team,

We are running self-hosted renovate because we are Bitbucket customers. Based on the [documentation](https://docs.renovatebot.com/self-hosting/#githubcom-token-for-release-notes) we need to create a personal access token on GitHub and it needs read-only access.

What is the minimum required scope to retrieve public repos release notes?

Thank you!

Scope list:
<img width="611" alt="Screen Shot 2021-07-21 at 21 04 08" src="https://user-images.githubusercontent.com/51825189/126545402-0788324f-0998-45b8-bed8-b349c2a276bb.png">

**Comment by viceice** [comment, marked as answer]
No scopes == readonly 😉

**Reply by andormarkus** [reply]
Thank you @viceice it worked
